src/main/java/com/modrinth/minotaur/TaskModrinthSyncBody.java

```diff
@@ -23,16 +24,16 @@
  * A task used to communicate with Modrinth for the purpose of syncing project body with, for example, a README.
  */
 public class TaskModrinthSyncBody extends DefaultTask {
-    /**
-     * Constant gson instance used for serializing the API request.
-     */
-    private static final Gson GSON = new Gson();
-
     /**
      * The extension used for getting the data supplied in the buildscript.
      */
     private final ModrinthExtension extension = getProject().getExtensions().getByType(ModrinthExtension.class);
 
+    /**
+     * Constant gson instance used for deserializing the API responses when files are uploaded.
+     */
+    private final Gson GSON = extension.getPrettyPrint().get() ? new GsonBuilder().setPrettyPrinting().create() : new Gson();
```

Comment:
Might as well make pretty printing the default, don't even need to have a separate config option for this.

Reply:
updated to 
```java
private final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
```